src/libcollections/slice.rs

```diff
@@ -437,8 +437,8 @@ impl<T> [T] {
     /// The caller must ensure that the slice outlives the pointer this
     /// function returns, or else it will end up pointing to garbage.
     ///
-    /// Modifying the slice may cause its buffer to be reallocated, which
-    /// would also make any pointers to it invalid.
+    /// Modifying the container this slice references may cause its buffer
```

Comment:
Hm, I wonder if "container" is the right word here, is an array a "container"? we don't really use it as a standard term, idk